text_to_image/README.md

```diff
@@ -29,7 +29,7 @@ export MODEL_PATH=$PWD/inference/text_to_image/model/
 ```
 ### Clone the repository
 ```bash
-git clone --recurse-submodules https://github.com/mlcommmons/inference.git --depth 1
```

Comment:
This path does not exist, it looks like a combination of the path one would use for ssh & https connection that ends up just throwing a 404